time stamp
uint16[] id_list

bool[] torque_enable
string[] drive_mode
string[] operation_mode
